src/bcrypt.cc

```diff
@@ -229,7 +211,19 @@ bcrypt(const char *key, const char *salt, char *encrypted)
 	/* We dont want the base64 salt but the raw data */
 	decode_base64(csalt, BCRYPT_MAXSALT, (u_int8_t *) salt);
 	salt_len = BCRYPT_MAXSALT;
-	key_len = strlen(key) + (minor >= 'a' ? 1 : 0);
+	if (minor <= 'a')
```

Comment:
use `{`

Reply:
It's OpenBSD source code, mostly modified. They prefer tabs and miss braces. Changing them will make further diffing with their source code impossible, as we have certain NodeJs and windows specific changes